tests/integration/integration-tests.sh

```diff
@@ -99,7 +101,7 @@ setup_test_pod() {
         break
     elif [ "${i}" -le "5" ]
     then
-        echo "HERE"
+        echo "Waiting for the pod to come online..."
```

Comment:
Perhaps you could make the sleep time exponential using `expr`, to allow more time for the pod to come online. Something like `SLEEP_TIME=$(expr $SLEEP_TIME \* 2); sleep $SLEEP_TIME`. Having first set `SLEEP_TIME=5` before the loop

Reply:
Cool! I like it! This is currently not really too much of an issue, in the sense that the pod generally comes online in 1 sec or so, but I mean, why not? it is definitely a cooler and more professional to do an exponential backoff